src/tools/time_governor.cc

```diff
@@ -48,8 +49,9 @@ const Tuple & TimeGovernor::get_input_time_type(double lower_bound, double upper
     return Tuple("TimeValue", "A time with optional unit specification.")
         .declare_key("time", Double(lower_bound, upper_bound), Default::obligatory(),
                                     "The time value." )
-		.declare_key("unit", String(), Default::read_time("Common time unit of the equation's Time Governor."),
-				"The time unit. Possible values: 's' seconds, 'min' minutes, 'h' hours, 'd' days, 'y' years.")
+		.declare_key("unit", UnitConverter::get_input_type(),
+                Default::read_time("The time unit."),
```

Comment:
Keep previous default value description. Related is `TimeGovernor.common_time_unit` that has to be changed as well.